src/libs/actions/TransactionEdit.ts

```diff
@@ -16,24 +16,24 @@ function createBackupTransaction(transaction: OnyxEntry<Transaction>) {
     };
 
     // Use set so that it will always fully overwrite any backup transaction that could have existed before
-    Onyx.set(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transaction.transactionID}`, newTransaction);
+    Onyx.set(`${ONYXKEYS.COLLECTION.TRANSACTION_BACKUP}${transaction.transactionID}`, newTransaction);
```

Comment:
This function, `createBackupTransaction`, is also used in `IOURequestStepAmount.tsx` - so i believe we should also be updating that component to subscribe to `ONYXKEYS.COLLECTION.TRANSACTION_BACKUP`

This looks like it's causing this blocker: https://github.com/Expensify/App/issues/40543

Reply:
Have a fix up here: https://github.com/Expensify/App/pull/40605